tools/downloader/pytorch_to_onnx.py

```diff
@@ -11,6 +11,18 @@
 import torch.onnx
 
 
+INPUT_DTYPE_TO_TORCH = {
+        'float': torch.float,
+        'half': torch.half,
+        'double': torch.double,
+        'uint8': torch.uint8,
+        'int8': torch.int8,
+        'int': torch.int32,
```

Comment:
```suggestion
        'int32': torch.int32,
```

I see no reason to deviate from the PyTorch naming in this one case.

Reply:
renamed